de.dlr.sc.virsat.external.lib.swagger/META-INF/MANIFEST.MF

```diff
@@ -0,0 +1,15 @@
+Manifest-Version: 1.0
+Bundle-ManifestVersion: 2
+Bundle-Name: VirSat External Lib - Swagger
+Bundle-SymbolicName: de.dlr.sc.virsat.external.lib.swagger
+Bundle-Version: 4.12.1.qualifier
+Bundle-RequiredExecutionEnvironment: JavaSE-1.8
+Bundle-Vendor: DLR (German Aerospace Center)
+Require-Bundle: de.dlr.sc.virsat.external.lib,
+ javax.servlet;visibility:=reexport,
+ de.dlr.sc.virsat.external.lib.jersey;bundle-version="4.12.1"
```

Comment:
Do we need this dependency? If we do, let's remove the version.

Reply:
No we don't, thanks :) This was needed when we had more swagger dependencies, but now we don't need it anymore!